src/wp-includes/interactivity-api/interactivity-api.php

```diff
@@ -66,7 +66,11 @@ function wp_interactivity_process_directives_of_interactive_blocks( array $parse
 
 	return $parsed_block;
 }
-add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks' );
+/*
+ * Uses a priority of 20 to ensure that other filters can add additional attributes to
+ * $parsed_block before the processing starts.
+ */
+add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks', PHP_INT_MAX, 1 );
```

Comment:
Please DO NOT use `PHP_INT_MAX`. There are many valid reasons why folks might want to filter block data after WP Core has finished it's processing, but before the block is rendered. Setting a priority that is higher than the default, along with the inline comment you've provided, should be enough to guard against most race conditions. If a much higher priority is necessary, use something like 99, which is used in a couple other places.

Reply:
@joemcgill 
> There are many valid reasons why folks might want to filter block data after WP Core has finished it's processing, but before the block is rendered.

What is a valid reason?